odoo/modules/loading.py

```diff
@@ -210,6 +210,7 @@ def load_test(idref, mode):
 
         idref = {}
 
+        env = False
```

Comment:
This is crappy code for a pseudo-optimization, as `api.Environment(cr, SUPERUSER_ID, {})` returns the same environment by design.